ddtrace/contrib/kafka/patch.py

```diff
@@ -203,3 +231,18 @@ def traced_commit(func, instance, args, kwargs):
     core.dispatch("kafka.commit.start", [instance, args, kwargs])
 
     return func(*args, **kwargs)
+
+
+def serialize_key(instance, topic, key, headers):
+    if _SerializationContext is not None and _MessageField is not None:
+        ctx = _SerializationContext(topic, _MessageField.KEY, headers)
+        if hasattr(instance, "_key_serializer") and instance._key_serializer is not None:
+            try:
+                key = instance._key_serializer(key, ctx)
+                return key
+            except Exception:
+                log.warning("Failed to set Consumer key tag: ", key, exc_info=True)
```

Comment:
We don't need to include a traceback in this warning. Also would could warning be noisy? If so, we should probably make it debug level

```suggestion
                log.warning("Failed to set Consumer key tag: %s", str(key))
```